### DISCUSSION TITLE
empty measures in XML output

### DISCUSSION THREAD

**Discussion by garyphilip** [Audiveris/audiveris#651] [Q&A]
I'm having no luck figuring out or finding answers online as to why I am getting empty measures such as this:

`    <measure number="2" width="143"></measure>
`

My first page had 52 measures, none empty. This portion of the 2nd page is mostly empty. The 9th measure has notes but I can't see what's different about it.

What are some things I might look for?

What do I need to post here to get help?

Thanks!

![Image13](https://github.com/Audiveris/audiveris/assets/42025604/f07e0d1a-6a1d-495d-9f8c-c30b5673a098)

**Comment by garyphilip** [comment]
The problem went away when I imported pages from a PDF as a book instead of PNGs for each page. I'd still like to know why this happened, as I may not have the PDF option in future, but it's not currently in my way.

**Comment by hbitteur** [comment]
> What do I need to post here to get help?

The [issues forum](https://github.com/Audiveris/audiveris/issues) is meant for issues like yours, while the discussion forum you have used is meant for rather general topics.

Anyway, could you post the .png input file for your 2nd page (the one you had problems with), so that we can investigate further.
